src/quo2/components/loaders/skeleton_list/component_spec.cljs

```diff
@@ -0,0 +1,30 @@
+(ns quo2.components.loaders.skeleton-list.component-spec
+  (:require [quo2.components.loaders.skeleton-list.view :as skeleton-list]
+            [quo2.foundations.colors :as colors]
+            [test-helpers.component :as h]))
+
+(defn test-skeleton
+  [content animated? i]
+  (let [animation-state (if animated? "animated" "static")
+        rendered        (h/render [skeleton-list/view
+                                   {:index         0
+                                    :content       content
+                                    :color         colors/neutral-10
+                                    :parent-height 600
+                                    :animated?     animated?}])
+        label-text      (str "skeleton-" animation-state "-" (mod i 4) "-" i)]
```

Comment:
maybe we should just change `label-text` to `accessibility` since that's what it actually is? 🤔

Reply:
I meant `accessibility-label` :)